bin/blocks-plot

```diff
@@ -131,16 +133,17 @@ if __name__ == "__main__":
     logging.basicConfig()
 
     parser = argparse.ArgumentParser()
-    parser.add_argument("--list", "-l", action="store_true", 
+    parser.add_argument("--list", "-l", action="store_true",
         help="List all channels contained in the experiments and exit")
-    parser.add_argument("--channels", "-c", type=str, 
+    parser.add_argument("--channels", "-c", type=str,
         help="A comma separated list of channels to plot. " + \
              "The wildcards '*' and '?' can be used to select multiple " + \
              "channels at once. Prefix a channel name with <num>: to " + \
              "refer to a channel of a particular experiment")
-    #parser.add_argument("--xaxis", "-x", choices=['iterations', 'epochs', 'time'],
-    #    default='iterations', help="Label/scale to use on the x-axis of the plot")
-    parser.add_argument("--ipython", action="store_true", 
+    parser.add_argument("--every", "-e",
+        choices=["epochs", "iters"], default="iters",
```

Comment:
Since it's "every epoch" or "every iteration" I would make the choise `epoch` and `iter` instead of `epochs` and `iters`